pkg/operatorsource/factory.go

```diff
@@ -10,7 +10,9 @@ import (
 
 	"github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
 	"github.com/operator-framework/operator-marketplace/pkg/appregistry"
-	"github.com/operator-framework/operator-marketplace/pkg/operatorsource/phase"
+	r "github.com/operator-framework/operator-marketplace/pkg/operatorsource/phase"
```

Comment:
Any reasoning for using the alias `r`? It seems a little confusing. Also, what's the difference between this `phase` package and the new one?